roboto/api_types.py

```diff
@@ -267,34 +389,130 @@ class SuccessfulPayment:
 
 
 @dataclass(frozen=True)
-class MessageBase:
-    """Base data for a Telegram message."""
+class PassportData:
+    """Contains information about
+    Telegram Passport data shared with the bot by the user."""
+
+    data: List[EncryptedPassportElement]
+    credentials: EncryptedCredentials
+
+
+@dataclass(frozen=True)
+class PassportFile:
+    """This object represents a file uploaded to Telegram Passport.
+    Currently all Telegram Passport files are in JPEG format when decrypted
+    and don't exceed 10MB."""
+
+    file_id: str
+    file_unique_id: str
+    file_size: int
+    file_date: int
+
+
+@dataclass(frozen=True)
+class EncryptedPassportElement:
+    """Contains information about
+    documents or other Telegram Passport elements shared with the bot by the user."""
+
+    type: str
+    hash: str
+    data: Optional[str] = None
+    phone_number: Optional[str] = None
+    email: Optional[str] = None
+    files: Optional[List[PassportFile]] = None
+    front_side: Optional[PassportFile] = None
+    reverse_side: Optional[PassportFile] = None
+    selfie: Optional[PassportFile] = None
+    translation: Optional[List[PassportFile]] = None
+
+
+@dataclass(frozen=True)
+class EncryptedCredentials:
+    """Contains information about
+    documents or other Telegram Passport elements shared with the bot by the user."""
+
+    data: str
+    hash: str
+    secret: str
+
+
+@dataclass(frozen=True)
+class InlineKeyboardMarkup:
+    """This object represents an inline keyboard
+    that appears right next to the message it belongs to."""
+
+    inline_keyboard: List[List[InlineKeyboardButton]]
+
+
+@dataclass(frozen=True)
+class InlineKeyboardButton:
+    """This object represents one button of an inline keyboard.
+    You must use exactly one of the optional fields."""
+
+    text: str
+    url: Optional[str] = None
+    login_url: Optional[LoginUrl] = None
+    callback_data: Optional[str] = None
+    switch_inline_query: Optional[str] = None
+    switch_inline_query_current_chat: Optional[str] = None
+    callback_game: Optional[CallbackGame] = None
+    pay: Optional[bool] = None
+
+
+@dataclass(frozen=True)
+class LoginUrl:
+    """This object represents a parameter of the
+    inline keyboard button used to automatically authorize a user."""
+
+    url: str
+    forward_text: Optional[str] = None
+    bot_username: Optional[str] = None
+    request_write_access: Optional[bool] = None
+
+
+@dataclass(frozen=True)
+class CallbackGame:
+    """A placeholder, currently holds no information."""
+
+
+@dataclass(frozen=True)
+class _MessageBase:
+    """Base data for a Telegram message.
+    This class is made this way to permit
```

Comment:
Please leave a blank line between the summary and the more complete description.